e2etest/get_stat_collection_test.go

```diff
@@ -0,0 +1,104 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2etest
+
+import (
+	"context"
+	"io/ioutil"
+	"path"
+	"runtime"
+	"testing"
+
+	pb "github.com/datacommonsorg/mixer/proto"
+	"github.com/datacommonsorg/mixer/server"
+	"github.com/google/go-cmp/cmp"
+	"google.golang.org/protobuf/encoding/protojson"
+	"google.golang.org/protobuf/testing/protocmp"
+)
+
+func TestGetStatCollection(t *testing.T) {
+	ctx := context.Background()
+
+	memcacheData, err := loadMemcache()
+	if err != nil {
+		t.Fatalf("Failed to load memcache %v", err)
+	}
+
+	client, err := setup(server.NewMemcache(memcacheData))
+	if err != nil {
+		t.Fatalf("Failed to set up mixer and client")
+	}
+	_, filename, _, _ := runtime.Caller(0)
+	goldenPath := path.Join(
+		path.Dir(filename), "../golden_response/staging/get_stat_collection")
+
+	for _, c := range []struct {
+		parentPlace string
+		childType   string
+		date        string
+		statVar     []string
+		goldenFile  string
+	}{
+		{
+			"geoId/06",
+			"County",
+			"2016",
+			[]string{"Count_Person", "Median_Age_Person"},
+			"CA_County_2016.json",
+		},
+		{
+			"country/USA",
```

Comment:
@pradh don't seem to have data for these two cases. Do we restrict on the the place types?

Reply:
per offline discussion, will add the support for not direct-linked place types in later PR